config/base/kustomization.yaml

```diff
@@ -10,7 +10,7 @@ patchesStrategicMerge:
 images:
 - name: controller
   newName: 957583890962.dkr.ecr.us-east-1.amazonaws.com/amazon-sagemaker-operator-for-k8s
-  newTag: latest
+  newTag: v1
```

Comment:
man.. a v2 update is going to be a lot of build system work.

Reply:
It has all been documented in the MCM.

Reply:
Thanks!